spec/controllers/api/v1/zones_spec.rb

```diff
@@ -0,0 +1,150 @@
+require "rails_helper"
+
+RSpec.describe "Api::V1::Zones", type: :request do
```

Comment:
```suggestion
RSpec.describe Api::V1::Zones, type: :controller do
```
E dùng type controller để test theo action nhé